pppd/plugins/Makefile.linux

```diff
@@ -30,7 +30,7 @@ all:	$(PLUGINS)
 	for d in $(SUBDIRS); do $(MAKE) $(MFLAGS) -C $$d all || exit $$?; done
 
 %.so: %.c
-	$(CC) -o $@ $(LDFLAGS) $(CFLAGS) $^
+	$(CC) -o $@ $(LDFLAGS) $(LDFLAGS_SHARED) $(CFLAGS) $^
```

Comment:
LDLIBS should be needed too.

Reply:
Did you mean LIBS? But it's for plugins (i.e. private non public DSOs), isn't enough to have LIBS in the final linkage of the executable?

Reply:
LIBS is non standard, the correct one is LDLIBS according to the doc:

https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html